### DISCUSSION TITLE
Cake.Unity - How to update info page?

### DISCUSSION THREAD

**Discussion by codewing** [cake-build/cake#3928] [Extension Q&A] [answered]
I'm using cake.unity (https://github.com/cake-contrib/Cake.Unity) and it lists the latest version as 0.8.1 but the package maintainer has released a 0.9.0 quite some time ago.
You can already see this in nuget here: https://www.nuget.org/packages/Cake.Unity

What system needs to be triggered to see this change?

**Comment by codewing** [comment, marked as answer]
Okay nevermind - it's just the website that isn't up to date...

**Comment by codewing** [comment]
So the question would be how to update this site?
https://cakebuild.net/extensions/cake-unity/

**Comment by pascalberger** [comment]
There's a tool automatically detecting updates and creating pull requests. Here's the pull request for Cake.Unity 0.9.0: https://github.com/cake-build/website/pull/2450/files.

Seems like subsequent deployment was not successful. A new deployment of the website is currently running and should be live in a few minutes.

**Reply by codewing** [reply]
Not quite sure I'll get it but sounds like this should work in the future, thanks a lot!

**Reply by pascalberger** [reply]
In more detail: There's the [Cake Addin Discoverer Tool](https://github.com/cake-contrib/Cake.AddinDiscoverer) which runs every few ours and checks for new or updated Cake addins published to nuget.org. The Cake website is a static website, which has metadata for all extensions stored in a repo (e.g. for Cake.Unity https://github.com/cake-build/website/blob/master/extensions/Cake.Unity.yml). If Cake Addin Discoverer finds new or updated addins it will create a pull request to updated these metadata. 
Once these PRs are reviewed & merged a new version of the website will be built and once this new version is deployed, information on the website is updated.

**Reply by codewing** [reply]
Thanks for taking the time and the detailed explanation! Now I got it!
